shared/utility/support/hostname.hpp

```diff
@@ -0,0 +1,13 @@
+#ifndef UTILITY_SUPPORT_HOSTNAME_HPP
+#define UTILITY_SUPPORT_HOSTNAME_HPP
+
+#include <string>
+#include <unistd.h>
+
+namespace utility::support {
+
+    std::string getHostname();
```

Comment:
I feel like you could just move the entire function into here
```suggestion
    inline std::string getHostname() {
        char* hostname = std::getenv("ROBOT_HOSTNAME");

        if (hostname == nullptr) {
            char buffer[255];
            ::gethostname(buffer, 255);    
            hostname = buffer;
        }

        return std::string(hostname);            
    }
```

Reply:
Yeah, was wondering that too